### PR TITLE
add `suspend_while_paused` option to `Kodi` check

### LINKED ISSUE #59: system not suspending while Kodi is paused
The default `Kodi` check keeps the system alive while Kodi playback is paused. Obviously, there are different use cases that might be interesting here. Some users might want to keep the system running while media is paused, where others (like me) would prefer to suspend it.

My actual use case is that when I switch inputs on my tv, Kodi pauses playback. But if I never switch back and actively **stop** playback, then the system will never suspend. To fix this, I added a new check that queries the `Player.Playing` condition.

Before submitting this as a PR, I'm wondering if you'd prefer this is a separate activity like it currently is or simply add it as an option to the existing `Kodi` activity (something like `suspendWhenPaused`, defaulting to `false` to stay backwards compatible with the current check)? I prefer the latter, since it does not really make sense to have a completely separate check class for this.

### PR DESCRIPTION
**Pull request by nioncode**
Fixes #59.

I also added some tests, but didn't run them, because I'm not familiar with setting this up and just hope travis will report any issues.

### CODE REVIEW

File: src/autosuspend/checks/activity.py [hunk 1]
```diff
@@ -94,23 +94,47 @@ def check(self):
...
 class Kodi(NetworkMixin, Activity):
 
-    QUERY = '?request={"jsonrpc": "2.0", "id": 1, ' \
-            '"method": "Player.GetActivePlayers"}'
+    @classmethod
+    def collect_init_args(cls, config) -> Dict[str, Any]:
+        try:
+            args = NetworkMixin.collect_init_args(config)
+            args['suspend_while_paused'] = \
+                config.getboolean('suspend_while_paused', fallback=False)
```
Comment (languitar) [inline src/autosuspend/checks/activity.py]: Can you break this line after opening bracket? That removes the need for the trailing slash.

File: src/autosuspend/checks/activity.py [hunk 2]
```diff
@@ -94,23 +94,47 @@ def check(self):
...
+            return args
+        except ValueError as error:
+            raise ConfigurationError(
+                'Configuration error ' + str(error)) from error
```
Comment (languitar) [inline src/autosuspend/checks/activity.py]: Please use string formatting here.

Comment (nioncode) [reply src/autosuspend/checks/activity.py]: I copied this from `KodiIdleTime`, that's why it has no string formatting in the first place :) Fixed.

File: src/autosuspend/checks/activity.py (hunk 1 above)
Comment (nioncode) [reply src/autosuspend/checks/activity.py]: Fixed.

**Comment by nioncode** [comment]
Fixed and re-pushed.

File: src/autosuspend/checks/activity.py (hunk 2 above)
Comment (languitar) [reply src/autosuspend/checks/activity.py]: Thanks. There are some things that have changed over time and that I should adapt at some point in time ;)

**Pull request merged**